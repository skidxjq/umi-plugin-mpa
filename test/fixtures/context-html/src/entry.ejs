<% if (htmlWebpackPlugin.options.title) {%>
  <title><%= htmlWebpackPlugin.options.title %></title>
  <%}else{%>
  <title>Document</title>
  <%}%>
<div>hello world</div>
