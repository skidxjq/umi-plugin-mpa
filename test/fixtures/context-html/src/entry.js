import ReactVersion from 'react/lib/ReactVersion';
console.log(ReactVersion.startsWith('15'));
alert('foooo');
