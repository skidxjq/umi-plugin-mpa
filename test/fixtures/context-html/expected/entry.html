
  <title>entry</title>
  
<div>hello world</div>
<script type="text/javascript" src="/entry.js"></script>